catalog/monitors/k8s.yaml

```diff
@@ -626,3 +626,31 @@ k8s-pending-pods:
     #critical_recovery:
     #warning_recovery:
 
+k8s-metrics-server-low-proc-count:
+  name: '(k8s) Low metrics-server Process Count'
+  type: process alert
+  query: |
+    processes('').over('kube_deployment:metrics-server').rollup('count').last('5m') < 4
```

Comment:
please research how to parametrize `< 4`

Reply:
+1 on adding a parameter instead of hardcoding `4`.